code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -297,24 +336,44 @@
 	speed = 1
 	wander = FALSE
 	alert_on_spacing = FALSE
+	death_sound = 'sound/mecha/mechmove03.ogg'
+	loot = list() // Explodes, doesn't drop loot.
 
 /mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/space/Process_Spacemove(movement_dir = 0)
 	return TRUE
 
-
+/mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/space/death()
+	visible_message("<span class='warning'>[src] explodes!</span>")
+	playsound(loc, 'sound/items/timer.ogg', 30, 0)
```

Comment:
```suggestion
	playsound(loc, 'sound/items/timer.ogg', 30, FALSE)
```

Reply:
Just gonna add this to a commit since I don't have suggestion commit permissions